velox/common/base/tests/StatusTest.cpp

```diff
@@ -128,5 +128,23 @@ TEST(StatusTest, macros) {
   ASSERT_TRUE(didThrow) << "VELOX_CHECK_OK did not throw";
 }
 
+Expected<int> modulo(int a, int b) {
+  if (b == 0) {
+    return folly::makeUnexpected(Status::UserError("division by zero"));
```

Comment:
we could also provide macros like other project do, such as:

```
VELOX_CHECK(b == 0, "division by zero");
```

or since we already have VELOX_CHECK, maybe `VELOX_RETURN_IF`?